lib/rules/forbid-component-props.js

```diff
@@ -140,10 +169,48 @@ module.exports = {
         return false;
       }
 
+      function checkIsTagForbiddenByAllowList() {
+        return options.allowList.indexOf(tagName) === -1;
+      }
```

Comment:
this seems overkill to make into a function, especially if it's used once. let's just inline it.